definitions/media-roku_app/definition.yml

```diff
@@ -0,0 +1,29 @@
+domain: MEDIA
```

Comment:
I'm proposing a new Media domain which will, over time, contain at least the following entities:

- This Roku App
- Mobile Video Player (PR to come)
- [Web Video Players](https://github.com/newrelic-experimental/entity-synthesis-definitions/pull/241) (this PR needs updating)
- Ad Slots for Google Publisher Tags